### DISCUSSION TITLE
Is there a way to paste the content from clipboard?

### DISCUSSION THREAD

**Discussion by windyliu1118** [getgauge/taiko#2610] [General]
Currently, we have a link to copy link to clipboard, how can we use one taiko API to paste the copied link?
I tried to use the press API as below, but seems it not working.
`await press(["ControlRight", "v"])`
It will not paste the link text in the checkbox.

**Comment by saikrishna321** [comment]
https://github.com/getgauge/taiko/discussions/2604#discussioncomment-3013408
